### PR TITLE
Use dopple instead of jsonrpcproxy

### LINKED ISSUE #5324: Replace scripts/jsonrpcproxy.py with dopple
Dopple is ["maintained"](https://github.com/ethereum/dopple/issues/1) at: https://github.com/ethereum/dopple

It would make sense to import it as a git submodule or subtree and be used that way.

### PR DESCRIPTION
**Pull request by axic**
Fixes #5324.

### CODE REVIEW

**Comment by gumb0** [comment]
No idea either.

**Comment by axic** [comment]
@gumb0, @chfast mentioned that you use dopple for something. I think this docker image is still working after the change, but I haven't tried it.

**Comment by gumb0** [comment]
I use it for ethstats reporting; I can try it from this PR.

File: scripts/dopple.py [hunk 1]
```diff
@@ -0,0 +1 @@
+dopple/dopple.py
```
Comment (gumb0) [inline scripts/dopple.py]: does it skip command line arguments?

Comment (axic) [reply scripts/dopple.py]: This is a symlink, github shows it strangely.

**Comment by axic** [comment]
@gumb0 @chfast anything blocking this?

**Comment by chfast** [comment]
This is also used in docker images.

**Comment by axic** [comment]
@chfast @gumb0 so should we merge? 😉 Please feel free to merge in that case.

**Comment by chfast** [comment]
Does the code is exactly the same? If yes, we can merge it. If not I have to verify it works in docker images as they are used in the Hive testing.

**Comment by axic** [comment]
> Does the code is exactly the same?

I've updated the other repo to match the on here when I made the PR. If there were changes to jsonrpcproxy in this repo since then, those aren't mirrored.

**Pull request merged**